### ISSUE TITLE
Users need to know what Verify is and what the Verify journey entails

### ISSUE THREAD

**Issue by EUzkuraityte** [alphagov/verify-local-patterns#602]
Including evidence from Cambridgeshire. The same was observed in the user lab in Newcastle. @lizziebruce @sanjaypoyzer could we prepare the users better to the journey, like giving more explanation about Verify -> IDP -> council journey and involvement?
![screen shot 2017-06-19 at 10 40 23](https://user-images.githubusercontent.com/20285115/27279567-f2c614a4-54dc-11e7-8232-96daba3ec3fa.png)
![screen shot 2017-06-19 at 10 46 59](https://user-images.githubusercontent.com/20285115/27279570-f48c0708-54dc-11e7-86e9-754c89e47b6e.png)

**Comment by lizziebruce** [comment]
@EUzkuraityte The IDP page/comments related to it is really a Verify Hub issue. Explanatory info should be as contextually close as possible to the thing being explained, the evidence even refers to "previous screens" and the last 3 before the IDP list are all Verify hub. Tagging @anthonylord for his info. Please see above Ant.

However, we could try to enhance the overall journey via user expectations from the council pages by changing the prove identity page copy to:

**Option 1**
> - answer security questions from an independent, government-approved company – you’ll choose the company

**Option 2**
> - you’ll choose an independent, government-approved company to check your identity
> - you’ll need to answer some security questions

**Option 3**
> - you’ll choose an independent, government-approved company to ask you security questions

In terms of the design/interaction/content comments:

1. Continue button at top isn't best practice - users may try to click it without having chosen an option
2. The spacing is there because there is a lot of dense information to absorb on the page, so space is needed
3. Radio buttons current size are best practice for accessibility - larger clickable area
4. We discussed a hyperlink to Verify but decided it was not beneficial as it took users off the journey - the explanatory bullets are the solution to this and new copy above should add further clarity
5. Only very high quality photographs of documents are accepted so we did not want to encourage users to upload photos. Users may not understand the term high resolution or a technical description of the required photo quality.

**Comment by EUzkuraityte** [comment]
Many thanks for looking into this issue @lizziebruce I like option 2 most, but I'm happy to hear from Ant and Sanjay as well on how we could enhance user's understanding regarding the Verify journey.

**Comment by lizziebruce** [comment]
Agree Option 2 is good, here's how it would present in situ, with Option 1 for comparison. Not including Option 3 as it's too open ended.

Option 1
![screen shot 2017-06-20 at 11 23 42](https://user-images.githubusercontent.com/27814324/27328760-27864e38-55ab-11e7-9c5d-d612f1af5260.png)

Option 2
![screen shot 2017-06-20 at 11 22 20](https://user-images.githubusercontent.com/27814324/27328750-214457ea-55ab-11e7-8f6e-c01a3816ee79.png)

**Comment by anthonylord** [comment]
@lizziebruce @EUzkuraityte Apologies for the delay in responding.
We've done a lot of research on how to make the company picker better for users, examples include ranking the companies based on conversion rate, highlighting just one, and providing stats to help users make a choice like average time taken and even customer ratings and reviews. Ranking the companies worked best, but unfortunately if we implement that we have to get the contract changed and all IDPs have to agree with it. Highlighting one company didn't work as most users wanted to know why government favoured that one company, was the company paying government to be the promoted one? And the stats didn't work as they were misinterpreted, for example, some users thought the companies with the lowest success rate thought they were the best to go with as they were the strictest, they caught more fraudsters. Users don't realise that there's a chance they won't be verified. The problem we have with the company picker is Hick's Law, the more choice a person has, the longer it takes to make a decision: https://en.wikipedia.org/wiki/Hick%27s_law. And it's frustrating that we tell users in the hub that we're gonna ask some questions to filter out companies, but if the user has 2 docs and a mobile, no companies are filtered out. We're still working on this, always will be. For research on Tuesday we've added a 'Help me choose' link, and if a user clicks it, we will show the companies ranked in order of conversion rates. If this works, hopefully it's a loophole in the contract as we're still showing the companies randomised, but if a user actively asks for help, we show the companies ranked. We can't actually show the conversion rates because then the IDPs will know how their competition is doing.
As for giving more explanation about Verify before Verify, it's definitely worth researching again; it might work better in LA land than in central government land. Whenever we've mentioned 'companies' out of context, before users get to the hub, some users get anxious or concerned and so abandon before they even get to the Verify hub. Other users don't realise they can continue there and then to Verify and choose a company, we see them going to Google to try to find a company that can verify them. These are some of the reasons we use progressive disclosure in the hub, to slowly introduce users to the fact that they will have to go to a 3rd party private company website to get verified. The slides start with good news, ie why Verify is a good thing, then we introduce the bad news, ie the companies, then finally more good news, ie you only have to do it once and it's reusable. 
Hope this is helpful in understanding some of the reasons the Verify hub is the way it is, and why we don't tell users anything about Verify on GOV.UK start pages other than it will take about 15 minutes. When Verify first went live, we were able to tell users on start pages that they will need a UK passport and/or UK driving licence, it was so much easier back then...

**Comment by lizziebruce** [comment]
Thanks @anthonylord. It sounds like it's worth researching in LA land as going to GOV.UK pages is already a disconnect from users council site and lots of users balk at the company list i.e. are surprised/horrified to see it. We've observed a lot of users simply clicking through the introductory progressive disclosure screens, as they have already had to absorb a lot on the preceding council service pages. 
All please see also #588.

**Comment by EUzkuraityte** [comment]
I agree with @lizziebruce users get information fatigue and just click through the hub pages, mostly skimming over the content or not reading at all. I think the  proposed new copy in #588 works and I'd like to see how this in UR and how this affects the user journey.

It will also be interesting to see what the Hub Improvement Team come up with as they are currently trying to resolve the issue of better preparing users to the IDP journey.

**Comment by suryanaga** [comment]
Yeah, really interested to see how telling users they'll have to choose a company here tests. Just preparing them early on might make a lot of difference.

**Comment by anthonylord** [comment]
If users get information fatigue, surely the answer isn't to provide even more information?

**Comment by lizziebruce** [comment]
@anthonylord hi Ant, what we've observed is that local users click through the first 3 blue Hub screens. So this content solution is to pre-empt them doing that and give them a heads up in advance to manage their expectations, as we've been finding users going click click click through the blue screens and then rearing up when they reach the list of IDPs. 
Another answer may be to modify what's on the blue screens but I know a lot of UR and iterative content design has gone into them for central / direct Verify applications. 
Another content solution could be to present different Verify Hub content if the origin is a local service – is that something we can do?

**Comment by lizziebruce** [comment]
Please see PR #610 and also issue #588 for current solution

**Comment by lizziebruce** [comment]
@anthonylord @sanjaypoyzer  btw I think there's a bug in the first blue Verify screen when coming from the local service? 
![screen shot 2017-06-28 at 12 31 58](https://user-images.githubusercontent.com/27814324/27635213-0c61b22e-5bfe-11e7-942b-72ded8c16a54.png)
Not sure if this is a Verify Central thing or something VL've produced, the url is https://govuk-verify-loa1.herokuapp.com/about?requestId=parking-permit-argleton-new&userLOA=0&selection=true

2nd para could read something like:

> Argleton County Council uses Verify to make sure noone else is applying for your parking permit. Only you will be able to give the correct information on its specific checks.

**Comment by lizziebruce** [comment]
Some initial thoughts on how the Verify Hub intro blue screens could work for arrivals from local services...

https://docs.google.com/a/digital.cabinet-office.gov.uk/document/d/1FvUf8sT9BbfEe1PsgdBuCWsqs6xMbKs0JDlcDf0JX-A/edit?usp=sharing

**Comment by anthonylord** [comment]
@lizziebruce I've always thought it might be better to have a different hub for LA services, the Verify hub was designed for central gov services and may not make sense in local land. It is something we can do, but we'd have to have specific research findings that points to the present hub being problamatic for users of LA services. Do we have that? Users clicking through the hub screens is common across all services, it's difficult to make people read.

**Comment by anthonylord** [comment]
@lizziebruce Was that bug sorted?

**Comment by lizziebruce** [comment]
@anthonylord re alternative Hub, I've referenced all the user issues that arose from the central Hub part of the journey, this should be good user research evidence? @petegale would be able to feed in further on this. Interesting, I didn't realise having a different hub for LA services was an option. I'd certainly like to try that. 
Re bug, the copy is displaying correctly now, but the presentation isn't working:
![screen shot 2017-07-03 at 12 08 29](https://user-images.githubusercontent.com/27814324/27790657-553add1a-5fe9-11e7-9987-ce9a83a3d93a.png)

**Comment by petegale** [comment]
@anthonylord @lizziebruce Regarding evidence for a local specific hub - having observed user research for both local and central gov interactions, I can't point to any specific findings that say that users of local authority services experience a different set of issues than those of central government. That's not to say that these users would not benefit from greater customisation of the hub to their task, but I think that is probably the case regardless of the central / local split

**Comment by ingasiler** [comment]
UR findings from 11 July
![screen shot 2017-07-18 at 15 43 34](https://user-images.githubusercontent.com/19390851/28323147-e4431b7a-6bcf-11e7-9646-328b99a0cdd8.png)

**Comment by ingasiler** [comment]
UR findings from 11 July
![screen shot 2017-07-18 at 15 44 11](https://user-images.githubusercontent.com/19390851/28323179-f9aa2f26-6bcf-11e7-8908-53a143ff82cd.png)

**Comment by ingasiler** [comment]
UR findings from 11 July
![screen shot 2017-07-18 at 15 45 00](https://user-images.githubusercontent.com/19390851/28323232-15edd804-6bd0-11e7-96fd-bdb95a6f6fc3.png)
